### DISCUSSION TITLE
Additional floors system?

### DISCUSSION THREAD

**Discussion by le-kote** [space-wizards/space-station-14#24721] [Ideaguys]
Who doesn't want to be able to go up to the second floor? I think floors system will be a HUGE innovation both for mappers and players.

**Comment by aac3master** [comment]
i think it requires **full** engine (core) rewriting. Also many logical problems.

**Comment by eris-webserv** [comment]
This is called Multi-Z and they're working on it (but be patient for now, it'll come when it's working well enough)
